Package.swift

```diff
@@ -30,7 +30,7 @@ let package = Package(
     ],
     dependencies: [
         .package(url: "https://github.com/IBM-Swift/Kitura.git", from: "2.1.0"),
-        .package(url: "https://github.com/IBM-Swift/BlueCryptor.git", .upToNextMinor(from: "0.8.0")),
+        .package(url: "https://github.com/IBM-Swift/BlueCryptor.git", .upToNextMinor(from: "1.0.0")),
```

Comment:
We should drop the `.upToNextMinor` constraint here.

Also, as Kitura-net needs updating to BlueSocket / BlueSSLService 1.0.0 to be compatible with the updated BlueCryptor, you'll need to update the Kitura dependency to `2.3.0` (not yet tagged, but coming soon).